content/contributing/how/translations/contents.lr

```diff
@@ -1,5 +1,7 @@
 _model: page
 ---
+_template: translations.html
+---
```

Comment:
This removal was actually deliberate - the translations template only serves to add the list of translated languages to the body of the text. In the previous version, the body ended with a "We have translations for:" line, which flowed into this extra content; but with the new body content, it makes less sense.

I've removed the template from the translated versions of this page as well; so - we either need to restore it to all versions, or delete it everywhere and delete the template entirely.

Reply:
I see. Are these translations discoverable at all without those links from the translations template?

I updated the wording....but it's just a strange instruction to "find the page in the language" you want to update...

Reply:
ahh...there's a globe next to the Edit button.

Reply:
Yes - there's a pulldown at the top of the page (the globe icon) that lets you select one of the existing translations.

Reply:
Ok; Update the instructions to use the globe to choose a language prior to clicking edit.